guides/doc-Provisioning_Guide/topics/configuring-the-discovery-service.adoc

```diff
@@ -35,7 +35,7 @@ endif::[]
 
 The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
 
-ifdef::satellite[]
+ifndef::satellite[]
```

Comment:
You can't just switch `ifdef` to `ifndef`. Instead, you should remove it.

Reply:
Do you mean it's not applicable to any of the builds (foreman, katello ,etc.) ?

Reply:
Previously this section was only relevant to Satellite. That's what `ifdef::satellite[]` means: it only gets rendered in the Satellite build. Now we said that it no longer works for Satellite so it should be `if NEVER`, but then it's actually better to remove it.

Reply:
Ok. I was informed it may be applicable to other builds so went with it. Removing :+1: